include/involution2d_cuda.cuh

```diff
@@ -7,7 +7,7 @@
 namespace involution {
 namespace cuda {
 
-#define CUDA_MAX_THREADS 512u
+#define CUDA_MAX_THREADS 1024u
 
```

Comment:
In your CuPy implementation, the maximum number of CUDA threads was set to 1024. However, when I experimented, my CuPy implementation did not work with 1024, so I set it to 512.

My CUDA implementation does work with 1024. However, when I experimented, I set it to 512 and forgot to change it back to 1024.